modules/impact.feature.interact.gui.focus-gui.d.ts

```diff
@@ -17,9 +18,15 @@ declare global {
       canPlayFocusSounds(this: this): boolean;
       setPressed(this: this, pressed: boolean): void;
       setActive(this: this, active: boolean): void;
+      focusGained(this: this): void;
+      focusLost(this: this): void;
+      invokeButtonPress(this: this): void;
+      canPlayFocusSounds(this: this): boolean;
+      setPressed(this: this, pressed: boolean): void;
+      
```

Comment:
Duplicate? Also missing methods.

```suggestion
      canLeaveFocus(this: this): boolean;
      setPressed(this: this, pressed: boolean): void;
      setActive(this: this, active: boolean): void;
      isSameAs(this: this, other: ig.FocusGui): boolean;
```